docker-storage-setup.1

```diff
@@ -93,11 +93,39 @@ DEVICE_WAIT_TIMEOUT:
 WIPE_SIGNATURES:
       Wipe any signatures found on disk. Valid values are
       true/false and default value is false. By default if any
-      signatures are found on disk operation is aborted. If this value
-      is set to true, then signatures will either be wiped or
+      signatures are found on disk operation is aborted. If this
+      value is set to true, then signatures will either be wiped or
       overwritten as suitable. This also means that if there is any
       data on disk, it will be lost.
 
+DOCKER_ROOT_VOLUME:
+      By default no new volume and filesystem will be setup for docker
+      root dir. Docker creates /var/lib/docker/ on top of underlying
+      filesystem for storing images and containers.
+
+      To carve out a separate logical volume for storing docker
+      images/containers/volumes data, set DOCKER_ROOT_VOLUME=yes
+      NOTE: devicemapper stores images/containers on thin pool.
+
+      docker-storage-setup creates a logical volume with an XFS
+      filesystem mounted on docker root directory (default: /var/lib/docker).
```

Comment:
Do man `./docker-storage-setup.1` and make sure all the lines look fine. This line seems to wrap.